app/src/main/java/com/eveningoutpost/dexdrip/utils/Mdns.java

```diff
@@ -294,15 +331,40 @@ public void onResolveFailed(NsdServiceInfo serviceInfo, int errorCode) {
                 locked_until = 0;
             }
 
+            // Fix the host addresses returned for LocalLink and sitelocal ipv6 addresses. The address returned for them should
+            // be in the format IPv6-address%scope_id. See https://docs.oracle.com/javase/8/docs/api/java/net/Inet6Address.html
+            // for more details.
+            private String getFixedHostAddress(InetAddress host ) {
+                if(host instanceof Inet6Address){
+                    UserError.Log.d(TAG,"Resolved IPV6"+ host );
+                    Inet6Address i6 = (Inet6Address)host;
+                    if(i6.isLinkLocalAddress() || i6.isSiteLocalAddress()) {
+                        try {
+                            host = Inet6Address.getByAddress(null, host.getAddress(), getWifiScopeId());
+                            return host.getHostAddress();
+                        } catch (UnknownHostException e) {
+                            UserError.Log.e(TAG, "Error: Got exception in Inet6Address.getByAddress ipv6 will probably not work", e);
+                            // Fall back to the original behavior.
+                            return host.getHostAddress();
+                        }
+                    }
+                    UserError.Log.d(TAG,"Resolved IPV6 golbal addresses");
+                    return host.getHostAddress();
+                } else{
+                    UserError.Log.d(TAG,"Resolved IPV4");
+                    return host.getHostAddress();
+                }
+            }
+
             @Override
             public void onServiceResolved(NsdServiceInfo serviceInfo) {
 
                 final InetAddress host = serviceInfo.getHost();
-                final String address = host.getHostAddress();
-                UserError.Log.d(TAG, serviceInfo.getServiceName() + " Resolved address = " + address);
+                final String address = getFixedHostAddress(host);
+                UserError.Log.d(TAG, serviceInfo.getServiceName() + " Resolved address = " + address );
                 final String short_name = shortenName(serviceInfo.getServiceName().toLowerCase());
                 if (!address.contains(":") || (iplookup.get(short_name) == null) || (JoH.msSince(iplookup.get(short_name).received) > 60000)) {
-                    iplookup.put(short_name, new LookUpInfo(address, JoH.tsl(), serviceInfo));
+                    iplookup.put(short_name, new LookUpInfo(address, JoH.tsl(), serviceInfo ));
```

Comment:
There is no need to add a space before the last two brackets, also not two lines above.

Reply:
It looks like still no common code formatting is established in this project and many parts of the code is inconsistently formatted (i.e. function `getFixedHostAddress`).